lang/en/package.html

```diff
@@ -17,66 +17,61 @@ <h2 class="details-main--title d-inline-block m-2">…</h2>
             <!-- react-text: 8 -->
             <!-- /react-text -->
           </a><span class="ais-Hit--popular null" title="0 downloads in the last 30 days">0</span>
-          <!-- react-empty: 10 --><span class="ais-Hit--version">0.0.0</span></div>
+          <!-- react-empty: 10 --><span class="ais-Hit--version"></span></div>
         <p class="m-2 lead">…</p>
         <!-- react-empty: 13 -->
       </header>
       <section id="readme" class="details-doc">
+        <h3 class="details-doc--title details-doc--title__readme py-1"><a href="#readme">readme</a></h3>
       </section>
     </section>
     <aside class="details-side col-lg-4">
       <article class="details-side--links">
         <div class="detail-links">
           <a href="https://github.com//" class="details-links--link details-links--link__github"><img src="/assets/search/ico-github.svg" alt="">
-            <!-- react-text: 23 -->/
+            <!-- react-text: 22 -->/
             <!-- /react-text -->
           </a>
           <a href="https://www.npmjs.com/package/…" class="details-links--link details-links--link__npm"><img src="/assets/search/ico-npm.svg" alt="">
-            <!-- react-text: 26 -->…
+            <!-- react-text: 25 -->…
             <!-- /react-text -->
           </a>
         </div>
       </article>
       <article class="details-side--copy">
         <h1>Use it</h1>
-        <div class="copyable"><code class="copyable--code"><!-- react-text: 31 -->$ <!-- /react-text --><span>yarn add …</span></code><button class="copyable--button"><img src="/assets/detail/ico-copy-default.svg" alt="" class="copyable--button__img"><!-- react-text: 35 -->copy<!-- /react-text --></button></div>
+        <div class="copyable"><code class="copyable--code"><!-- react-text: 30 -->$ <!-- /react-text --><span>yarn add …</span></code><button class="copyable--button"><img src="/assets/detail/ico-copy-default.svg" alt="" class="copyable--button__img"><!-- react-text: 34 -->copy<!-- /react-text --></button></div>
         <div><a class="details-side--runkit" href="https://runkit.com/npm/…" target="_blank" rel="noopener noreferrer">Try in RunKit</a></div>
       </article>
       <article class="details-side--popularity">
         <h1>Popularity</h1>
         <dl></dl>
       </article>
-      <article class="details-side--activity">
-        <h1>Activity</h1>
```

Comment:
This doesn't show anymore in the detail page when there's no GitHub repo, so it doesn't show by default